DataProcessing/DB/dal.py

```diff
@@ -5,7 +5,8 @@
 from sqlalchemy.ext.declarative import declarative_base
 
 # DB_LOCATION = '/mnt/raid1/home/bar_cohen/Shoham_KG.db' ## NEVER CHANGE THIS !!!
-DB_LOCATION = '/mnt/raid1/home/bar_cohen/42Street.db' ## NEVER CHANGE THIS !!!
+DB_LOCATION_VAL = '/mnt/raid1/home/bar_cohen/42Street.db' ## NEVER CHANGE THIS !!!
+DB_LOCATION_TEST = '/mnt/raid1/home/bar_cohen/42Street_test.db' ## NEVER CHANGE THIS !!!
```

Comment:
I suggest we leave the constant DB_LOCATION and set it either to DB_LOCATION_VAL or DB_LOCATION_TEST based on the db we want to run against. Otherwise, if we will want to run again with the validation set we will need to change again all the db locations you changed below.